diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter3/MagicNumberRule.kt

```diff
@@ -108,7 +108,7 @@ class MagicNumberRule(configRules: List<RulesConfig>) : DiktatRule(
     companion object {
         const val IGNORE_TEST = true
         const val NAME_ID = "aca-magic-number"
-        val ignoreNumbersList = listOf("-1", "1", "0", "2")
+        val ignoreNumbersList = listOf("-1", "1", "0", "2", "0U", "1U", "2U", "-1L", "0L", "1L", "2L", "0UL", "1UL", "2UL")
```

Comment:
hope that we really compare strings in the logic

Reply:
I had to change the logic a bit